Makefile

```diff
@@ -1,8 +1,8 @@
-.PHONY: all static clean
+.PHONY: all deps static clean
 
 # If you can use Docker without being root, you can `make SUDO= <target>`
 SUDO=sudo
-
+DOCKER_SQUASH=$(shell which docker-squash)
```

Comment:
Its a new tool which squashes down docker layers.  I've added it to make deps to install it for you.